.pre-commit-config.yaml

```diff
@@ -0,0 +1,24 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
```

Comment:
Newer version is available. See the same file in pumpkin-fun repo. Please edit to `v4.3.0`.

Reply:
Updated